stix_shifter_modules/cbcloud/stix_transmission/error_mapper.py

```diff
@@ -0,0 +1,24 @@
+from stix_shifter_utils.utils.error_mapper_base import ErrorMapperBase
+from stix_shifter_utils.utils.error_response import ErrorCode
+from stix_shifter_utils.utils import logger
+
+
+class ErrorMapper():
+
+    DEFAULT_ERROR = ErrorCode.TRANSMISSION_MODULE_DEFAULT_ERROR
+    logger = logger.set_logger(__name__)
+
+    @classmethod
+    def set_error_code(cls, json_data, return_obj):
+        error_code = cls.DEFAULT_ERROR
+
+        try:
+            # Carbon Black Cloud stores error messages in 'translation_key'
+            error_code = json_data['translation_key']
```

Comment:
is `translation_key` object contains the error codes? if you have the list of error codes you can create a dictionary and map the error codes just like we have in other modules.

Reply:
I do not have a mapping for these unfortunately.